cogs/fun.py

```diff
@@ -34,11 +35,20 @@ def find_message(score: float, ship_messages: List[object]) -> str:
 def calculate_score(first: discord.Member, second: discord.Member, ship_messages: List[object]) -> Tuple[float, str]:
     first_id: int = first.id
     second_id: int = second.id
+    
     if first_id == second_id:
         return 100, 'You\'re a perfect match... for yourself!'
-    else:
-        score = ((first_id + second_id) // 7) % 100
-        return score, find_message(score, ship_messages)
+
+    rigged = os.getenv(f'RIGGED_{first_id}')
+    if rigged is not None and str(second_id) == rigged:
+        return 100, 'It\'s a perfect match! That is very rare, so you should treasure it!'
+
+    rigged = os.getenv(f'RIGGED_{second_id}')
+    if rigged is not None and str(first_id) == rigged:
+        return 100, 'It\'s a perfect match! That is very rare, so you should treasure it!'
+
+    score = ((first_id + second_id) // 7) % 100
+    return score, find_message(score, ship_messages)
```

Comment:
Does something like this not work? Way less duped code
```suggestion
    rigged = os.getenv(f'RIGGED_{first_id}')
    rigged = rigged if rigged is not None else os.getenv(f'RIGGED_{second_id}')
    
    if rigged == str(first_id) or rigged == str(second_id):
        score = 100
    else:
        score = ((first_id + second_id) // 7) % 100

    return score, find_message(score, ship_messages)
```